pkg/machine/hyperv/machine.go

```diff
@@ -683,3 +693,29 @@ func (m *HyperVMachine) setRootful(rootful bool) error {
 	m.HostUser.Modified = true
 	return nil
 }
+
+func (m *HyperVMachine) cleanupGVProxy() {
+	gvPid, err := m.GvProxyPid.Read()
+	if err != nil {
```

Comment:
Can we return errors in this function but let the call sites decide whether to suppress them or not?

Reply:
fixed